pallets/ddc-nodes/Cargo.toml

```diff
@@ -23,10 +25,12 @@ default = ["std"]
 std = [
   "codec/std",
   "ddc-primitives/std",
+  "frame-benchmarking/std",
   "frame-support/std",
   "frame-system/std",
   "scale-info/std",
   "sp-runtime/std",
   "sp-std/std",
   "sp-core/std",
 ]
+runtime-benchmarks = ["frame-benchmarking/runtime-benchmarks"]
```

Comment:
Every frame dependency that has the runtime-benchmarks feature should be added to this list so that when compiling with that feature, compilation does not fail because of missing benchmarking-specific functionality from those components.